tests/test_popstepsampling.py

```diff
@@ -54,6 +54,25 @@ def test_stepsampler_cubegausswalk(plot=False):
     assert a.sum() > 1
     assert b.sum() > 1
 
+def test_stepsampler_randomEllSlice(plot=False):
```

Comment:
please also add a unit test of `update_vectorised_slice_sampler`, if it does not exist yet